pubsublite/manager.go

```diff
@@ -101,13 +105,22 @@ func (m *Manager) ListReservations(ctx context.Context) ([]string, error) {
 			}
 			return nil, fmt.Errorf("pubsublite: failed listing reservations: %w", err)
 		}
-		names = append(names, path.Base(reservation.Name))
+		reservationName := path.Base(reservation.Name)
+		if !strings.HasPrefix(reservationName, namespacePrefix) {
+			continue
+		}
+		names = append(names, reservationName[len(namespacePrefix):])
```

Comment:
Q: why are we returning names without prefix here?

Reply:
The idea is that the namespace should only ever be specified in one place, and only ever surfaced as a separate dimension (in metrics) or attribute (in logs and traces). That way one can, for example, compare metrics for a topic with the same name in two different namespaces.

Reply:
I see, I also read your updated comment. As PubSub does not support the concept of "namespace" we consider it an implementation detail so we hide it to callers. Thanks for the clarification.